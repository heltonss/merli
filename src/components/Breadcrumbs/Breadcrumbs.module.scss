@import '../../assets/scss/colors';
@import '../../assets/scss/mixins';

.link {
  @include fontSize(14px);
  display: inline-block;
  color: $dark-gray;
  padding: 16px 0;
  text-transform: lowercase;

  &::first-letter {
    text-transform: capitalize;
  }

  &:hover,
  :active {
    color: $dark-gray;
  }

  &::after {
    position: relative;
    content: '';
    display: inline-block;
    width: 0.4em;
    height: 0.4em;
    border-right: 1px solid $dark-gray;
    border-top: 1px solid $dark-gray;
    transform: rotate(45deg);
    margin: 0 0.6em;
  }

  &:last-child {
    font-weight: bolder;
    &::after {
      content: '';
      display: none;
    }
  }
}
