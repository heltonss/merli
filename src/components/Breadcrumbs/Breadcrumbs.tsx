import React from 'react';
import { Link } from 'react-router-dom';
import Style from './Breadcrumbs.module.scss';

const Breadcrumbs: React.FC = () => {
  return (
    <>
      <Link className={Style.link} to="/eletronica">
        eletronica
      </Link>
      <Link className={Style.link} to="/eletronica">
        eletronica 2
      </Link>
      <Link className={Style.link} to="/eletronica 3">
        eletronica duarte
      </Link>
      <Link className={Style.link} to="/eletronica 4">
        eletronica teste
      </Link>
    </>
  );
};

export default Breadcrumbs;
