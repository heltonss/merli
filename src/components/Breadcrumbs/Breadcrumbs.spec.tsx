import { shallow } from 'enzyme';
import React from 'react';
import Breadcrumbs from './Breadcrumbs';

describe('Breadcrumb', () => {
  it('should render withou crash', () => {
    const wrapper = shallow(<Breadcrumbs />);

    expect(wrapper).toMatchSnapshot();
  });
});
