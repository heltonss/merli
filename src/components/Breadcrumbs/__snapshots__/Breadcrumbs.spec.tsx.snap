// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Breadcrumb should render withou crash 1`] = `ShallowWrapper {}`;
